### PR TITLE
Feat config

### PR DESCRIPTION
**Pull request by jmdobry**
This PR uses #14 as its base.

### CODE REVIEW

**Comment by jmdobry** [comment]
@crobinson42 Thoughts?

File: src/handlers.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      .then((result) => {
+        req.jsdataResult = result
+        next()
+      })
+      .catch(next)
+  }
+}
+
+export function makeResponseHandler (method, component, config = {}) {
+  const statusCode = config[method].statusCode || DEFAULTS[method].statusCode
```
Comment (crobinson42) [inline src/handlers.js]: This will throw `Uncaught TypeError: Cannot read property 'statusCode' of undefined(…)` if `config[method]` is not defined.

**Comment by crobinson42** [comment]
looks great! Very readable in what's going on in the code/flow.

File: src/handlers.js (hunk 1 above)
Comment (jmdobry) [reply src/handlers.js]: Fixed.

**Pull request merged**